src/schema/types.ts

```diff
@@ -0,0 +1,121 @@
+import { Message } from "wechaty";
+
+export interface AdapterOptions {
+    /**
+     * The adapter's name.
+     * @type {string}
+     * @memberof AdapterOptions
+     **/
+    name: string;
+}
+
+export enum MessageType {
```

Comment:
Why do not import `MessageType` from `wechaty-puppet` module?

Reply:
Because I thought it was a strange thing to export the MT definition from a very deep path `wechaty-puppet/dist/esm/src/schemas/message`, so I copied that enumeration into the project's file.